multi-factor-auth.md

```diff
@@ -0,0 +1,20 @@
+# Multi-Factor Authentication
+
+Multi-factor authentication is required for any member on any platform where that member is acting with admin / moderator privleges.
+For example, members who have the Owner role on GitHub, or Organizer / Co-Organizer role on Discord, are required to enable multi-factor authentication
```

Comment:
Organizer / Co-Organizer role should just be `Organizers` to match what is on Discord.